src/Notifier.js

```diff
@@ -133,29 +153,35 @@ var Notifier = {
         }
 
         if(enable) {
-            if (!this.havePermission()) {
-                global.Notification.requestPermission(function() {
-                    if (callback) {
-                        callback();
+            // Case when we do not have the permission as 'granted'
+            if (this.isPermissionDefault()) {
```

Comment:
I don't think there's any need for this test. `global.Notification.requestPermission()` will do nothing (except give result==granted) if we already have permission.